core/src/main/java/com/sequenceiq/cloudbreak/converter/util/GatewayConvertUtil.java

```diff
@@ -0,0 +1,96 @@
+package com.sequenceiq.cloudbreak.converter.util;
+
+import java.security.KeyPair;
+import java.security.cert.X509Certificate;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import com.sequenceiq.cloudbreak.api.model.stack.cluster.gateway.GatewayJson;
+import com.sequenceiq.cloudbreak.api.model.stack.cluster.gateway.GatewayTopologyJson;
+import com.sequenceiq.cloudbreak.api.model.stack.cluster.gateway.SSOType;
+import com.sequenceiq.cloudbreak.client.PkiUtil;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.gateway.Gateway;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.gateway.GatewayTopology;
+
+@Component
+public class GatewayConvertUtil {
+
+    @Inject
+    private ConversionService conversionService;
+
+    public void setTopologies(GatewayJson source, Gateway gateway) {
```

Comment:
This method could be dropped and '''setTopologies''' be used which should be moved here from ClusterRequestToGatewayConverter

Reply:
Also this method does not provide the backward compatibility